Playbooks/playbook-Akamai_WAF_Network_list_activate_-_Generic.yml

```diff
@@ -0,0 +1,274 @@
+id: Akamai WAF Network list activate generic polling
```

Comment:
1) id and name should be title format.
2) no need to indicate implementation (generic polling)
`Akamai WAF - Activate Network Lists`